packages/manager/src/components/Link.stories.tsx

```diff
@@ -1,32 +1,67 @@
 import React from 'react';
 
+import DocsIcon from 'src/assets/icons/docs.svg';
 import { Link } from 'src/components/Link';
-import { Typography } from 'src/components/Typography';
 
 import type { Meta, StoryObj } from '@storybook/react';
-import type { LinkProps } from 'react-router-dom';
+import type { LinkProps } from 'src/components/Link';
 
-// TODO: remove the typography component from this story once M3-6772 is handled
+/**
+ * An instance of Link component given a relative URL, rendering a React Router `Link` component.
+ */
 export const Default: StoryObj<LinkProps> = {
+  render: (args: LinkProps) => <Link {...args}>{args.children}</Link>,
+};
+
+/**
+ * An instance of Link component given an absolute URL, rendering an `a` tag with `target="_blank"`.
+ * This version does not provide an `external` prop, usually meaning it belongs to the same domain or subdomain.<br />
+ * **Example**: a documentation link<br />
+ * It will open in a new tab since not being relative (internal).
+ */
+export const External: StoryObj<LinkProps> = {
+  name: 'External Link',
   render: (args: LinkProps) => (
-    <Typography variant="body1">
-      <Link {...args} to="/">
-        {args.children} (internal link)
-      </Link>
-    </Typography>
+    <Link
+      {...args}
+      to="https://www.linode.com/docs/products/compute/compute-instances/faqs"
+    >
+      https://www.linode.com/docs/products/compute/compute-instances/faqs
+    </Link>
   ),
 };
 
 /**
- * **@deprecated**<br />
- * This story is deprecated and will be removed in a future release.
- * Please use the `ExternalLink` component when you need to render an external link.
+ * An instance of Link component given an absolute URL, rendering an `a` tag with `target="_blank"`.
+ * This version provides an `external` prop, usually belonging to the same domain or a subdomain.<br />
```

Comment:
```suggestion
 * This version provides an `external` prop, to be used when the site does not belong to the same domain or a subdomain.<br />
```

Is this what we mean here? Otherwise, I'm a little confused.

Reply:
yes! you're right, fixed!